src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,67 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.IntStream;
+import racingcar.utils.RandomUtils;
+import racingcar.view.OutputView;
+
+public class Cars {
+
+    private final List<Car> cars = new ArrayList<>();
+
+    public Cars(final List<Name> names) {
+        validateNames(names);
+        for (Name name : names) {
+            cars.add(new Car(name, Position.ZERO));
+        }
+    }
+
+    public Cars(final List<Name> names, final List<Position> positions) {
+        validateNames(names);
+        IntStream.range(0, names.size())
+            .forEach(i -> cars.add(new Car(names.get(i), positions.get(i))));
+    }
+
+    public List<Car> getWinners() {
+        List<Car> winners = new ArrayList<>();
+
+        cars.stream()
+            .filter(it -> it.isWinner(getMaxDistance()))
+            .forEach(winners::add);
+        return Collections.unmodifiableList(winners);
+    }
+
+    public void move() {
+        for (Car car : cars) {
+            car.move(RandomUtils.getRandomNumber());
+            OutputView.printCarInformation(car);
+        }
+    }
+
+    private int getMaxDistance() {
+        int maxDistance = 0;
+        for (Car car : cars) {
+            maxDistance = Math.max(maxDistance, car.getDistance());
+        }
+        return maxDistance;
+    }
+
+    private void validateNames(List<Name> names) {
+        int originCount = names.size();
+        Set<Name> set = new HashSet<>();
+
+        for (Name name : names) {
+            set.add(name);
+        }
```

Comment:
```suggestion
        Set<Name> set = new HashSet<>(names);
```
요렇게 하면 한방에 가능합니다. Collection 끼리는 생성자를 통해 쉽게 서로 다른 Collection으로 변환할 수 있어요 :)

Reply:
좋은 팁 감사합니다!! 컬렉션에 대해 더 공부해볼게요 😃